project/gh-action/infrastructure/kubernetes/configmap.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: web-config
+  namespace: production
+data:
+  APP_ENV: "production"
+  CACHE_TTL: "3600"
+  API_VERSION: "v1"
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)